src/test/unit/marker-schema.test.js

```diff
@@ -256,6 +256,8 @@ describe('marker schema formatting', function () {
       ['bytes', 10],
       ['bytes', 12.3456789],
       ['bytes', 123456.789],
+      ['bytes', 123456789],
+      ['bytes', 123456789123],
```

Comment:
(no action needed:) It's interesting to me that `format-numbers.js` is tested via the marker schema. It would make more sense to me to have these tests in `format-numbers.test.js` but I guess you're just following what's already there.